LibsAndSamples.sdk.slnf

```diff
@@ -0,0 +1,22 @@
+{
+  "solution": {
+    "path": "LibsAndSamples.sln",
+    "projects": [
+      "ManagedIdentityTokenRevocation/ManagedIdentityTokenRevocation.csproj",
```

Comment:
please make sure to get the latest from main. This `ManagedIdentityTokenRevocation` was wrongly placed in this path and has been moved to the correct location now.

Reply:
Thanks!